storybook-example/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mosip/mosip-plugins"
+    "url": "git+https://github.com/mosip/mosip-plugins/storybook-example"
```

Comment:
remove the /storybook-example from the link. Link is not valid

Reply:
resolved in this [ca7c463](https://github.com/mosip/mosip-plugins/pull/56/commits/ca7c463b799a7e2584b2318c9bd15a2fa66880e7)